content/typescript/concepts/promises/promises.md

```diff
@@ -0,0 +1,56 @@
+| Title    | Description                                                                                                            | Subjects           | Tags                             | CatalogContent                   |
```

Comment:
@felpsg take a look at the documentation folder where there are templates for the front matter: https://raw.githubusercontent.com/Codecademy/docs/main/documentation/concept-entry-template.md